Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3
+MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
+
+RUN apt-get update
+
+RUN mkdir -p /home/Bassa
+RUN apt-get install git
+RUN git clone https://github.com/scorelab/Bassa.git /home/Bassa
+RUN chmod 755 /home/Bassa/setup.sh
+RUN /home/Bassa/setup.sh
```

Comment:
One good practice writing dockerfiles is combining the RUN commands together if they relate. 

You can do as follows, 

```
RUN apt-get update \
&&  chmod 755 /home/Bassa/setup.sh && \
&&  /home/Bassa/setup.sh && \
&&  python components/core/setup.py develop
```

Refer this : https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34